kernel/retroknowledge.mli

```diff
@@ -148,20 +144,18 @@ val find : retroknowledge -> field -> entry
 type reactive_info = {(*information required by the compiler of the VM *)
   vm_compiling :
       (*fastcomputation flag -> continuation -> result *)
-      (bool->Cbytecodes.comp_env->constr array ->
-       int->Cbytecodes.bytecodes->Cbytecodes.bytecodes)
+      (bool -> Cinstr.lambda array -> Cinstr.lambda)
```

Comment:
Since you are modifying the signature of this function and the following ones, don't you want to drop the `bool` argument? There has never been any way to set it to `false`.

Reply:
Honestly, the whole retroknowledge architecture is an example of premature generalization. It is all about hooks that can be instantiated only to one function, types that can be concretized in exactly one way, and booleans that can have only one value. All this will go away with the next patch introducing primitive integers, but for the scope of this PR, I'd prefer to leave things as they were.

Reply:
Fine by me.